anti-bruteforce-services/pom.xml

```
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>anti-bruteforce</artifactId>
+        <groupId>org.exoplatfrom</groupId>
+        <version>1.0.x-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>anti-bruteforce-services</artifactId>
```

Could you add the name tag here ?
```suggestion
    <artifactId>anti-bruteforce-services</artifactId>
    <name>eXo Anti-bruteforce - Services</name>
```

---

done